bitcampHomeworkWeekend05/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>bitcampHomeworkWeekend05</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+	</natures>
+</projectDescription>
```

Comment:
Obavezno dodati .gitignore fajl da se na repozitorij ne postavljaju fajlovi koji su specificni za okruzenje (morali ste spominjati to kad ste ucili Git, detalje procitaj ovdje: http://git-scm.com/docs/gitignore). Primjer jednog .gitignore fajla za Eclipse projekat mozes pogledati na ovoj zanimljivoj stranici koja generise njegov sadrzaj u zavisnosti od okruzenja: https://www.gitignore.io/api/eclipse. Takodjer, ukoliko koristis plugin za Eclipse tipa EGit, on bi trebao da izgenerise ovaj fajl automatski. Morat ces izbrisati sve nepotrebne fajlove sa repozitorija rucno kad dodas ovaj fajl.